sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -886,11 +887,18 @@ __SYCL_EXPORT bool is_source_kernel_bundle_supported(backend BE,
 /////////////////////////
 // syclex::create_kernel_bundle_from_source
 /////////////////////////
+
+__SYCL_EXPORT kernel_bundle<bundle_state::ext_oneapi_source>
+create_kernel_bundle_from_source(const context &SyclContext,
+                                 source_language Language,
+                                 const std::string &Source);
+
+#if (!defined(_HAS_STD_BYTE) || _HAS_STD_BYTE != 0)
 __SYCL_EXPORT kernel_bundle<bundle_state::ext_oneapi_source>
-create_kernel_bundle_from_source(
-    const context &SyclContext,
-    sycl::ext::oneapi::experimental::source_language Language,
-    const std::string &Source);
+create_kernel_bundle_from_source(const context &SyclContext,
+                                 source_language Language,
+                                 const std::vector<std::byte> &Bytes);
```

Comment:
@gmlueck I definitely think creating kernel bundles from spirv binary ( .spv ) is the right thing.  

But there is a tiny bit of a disconnect there, where we are creating "from source" and yes that source is .spv binary not .spt the textual human(*) readable representation of SPIR-V.  Anyway, I'm not trying to make more work for anyone. Nor do I think there is any real demand for using .spt sources.  But it does seem a bit funny.

(*) I guess .spt is "almost human" readable.

Reply:
I agree that calling SPIR-V binary "source" is a bit of a misnomer, but all the APIs work out pretty well.  We could easily add SPIR-V source support in the future by adding a new enumerator like `source_language::spirv_source`.  However, I think there is no compelling reason to do that unless there is some request.